src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -98,6 +111,16 @@ class IqPolicyEvaluatorUtil
     }
   }
 
+  private static boolean isHttp404Error(final Exception throwable) {
+    int httpResponseExceptionIndex = ExceptionUtils.indexOfType(throwable, HttpResponseException)
+    if (httpResponseExceptionIndex >= 0) {
+      Throwable[] throwables = ExceptionUtils.getThrowables(throwable)
+      HttpResponseException httpResponseException = (HttpResponseException) throwables[httpResponseExceptionIndex]
+      return httpResponseException.statusCode == 404
```

Comment:
Instead of hard coding we could use HttpStatus.NOT_FOUND_404

Reply:
> Instead of hard coding we could use HttpStatus.NOT_FOUND_404

Changed c37de51